tools/depends/target/libspdlog/Makefile

```diff
@@ -0,0 +1,67 @@
+-include ../../Makefile.include
+DEPS = Makefile
+
+# lib name, version
+LIBNAME=spdlog
+VERSION=1.5.0
+SOURCE=$(LIBNAME)-$(VERSION)
+ARCHIVE=$(SOURCE).tar.gz
+
+CMAKE_OPTIONS= \
+	-DSPDLOG_BUILD_EXAMPLE=OFF \
+	-DSPDLOG_BUILD_TESTS=OFF \
+	-DSPDLOG_BUILD_BENCH=OFF \
+	-DSPDLOG_INSTALL=ON \
+	-DSPDLOG_FMT_EXTERNAL=ON
+
+ifeq ($(CROSS_COMPILING), yes)
+  DEPS += ../../Makefile.include
+else
+  CXXFLAGS += -std=c++14
+  ROOT_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+
+  ifeq ($(PLATFORM),)
+    PLATFORM = native
+    TARBALLS_LOCATION = $(ROOT_DIR)
+    BASE_URL := http://mirrors.kodi.tv/build-deps/sources
+    RETRIEVE_TOOL := curl
+    RETRIEVE_TOOL_FLAGS := -Ls --create-dirs -f -O
+    ARCHIVE_TOOL := tar
+    ARCHIVE_TOOL_FLAGS := --strip-components=1 -xf
+    CMAKE := cmake -DCMAKE_INSTALL_PREFIX=$(PREFIX)
+  endif
+endif
+
+LIBDYLIB=$(PLATFORM)/build/libspdlog.a
+
+.PHONY: .installed-native
+
+all: .installed-$(PLATFORM)
+
```

Comment:
```suggestion
download: $(TARBALLS_LOCATION)/$(ARCHIVE)
```

Reply:
And what triggers the `download` target?

Reply:
this is mostly used for my packaging, only triggered manually and not used in the normal build process

Reply:
Ah so this is in addition to the `$(TARBALLS_LOCATION)/$(ARCHIVE)` target.

Reply:
Added